packages/forma-36-react-components/src/components/Button/Button.css

```diff
@@ -106,7 +106,7 @@
   }
 
   &.Button:link {
-    border-color: var(--color-blue-dark);
+    border-color: var(--color-blue-mid);
```

Comment:
I was not sure if I could just remove the `&.Button:link` block so I just changed the color